storage/bbolt/dedup.go

```diff
@@ -15,12 +15,12 @@
 // Package bbolt implements modules/dedup using BBolt.
 //
 // It contains two buckets:
-//   - The dedup bucket stores <leafID, idx> pairs. Entries can either be added after sequencing,
-//     by the server that received the request, or later when synchronising the dedup storage with
-//     the log state.
+//   - The dedup bucket stores <leafID, idx::timestamp> pairs. Entries can either be added after
```

Comment:
How about `... <leafID, {IDX, timestamp}> pairs`?
The `::` feels like it's implying some sort of dereference/scope relationship between idx and timestamp (probably because it's used like that in C++)

Reply:
Heheh, I'm coming from math + functional programing background, so `{A,B}` makes me thinks of Sets, and I borrowed `::` from Caml. I changed it to `{A,B}` I'm not too attached to this :)